aspnetcore/test/troubleshoot.md

```diff
@@ -61,3 +61,88 @@ This warning appears when the environment variable `PATH` doesn't point to any .
 
 * Install or verify the .NET Core SDK is installed.
 * Verify that the `PATH` environment variable points to the location in which the SDK is installed. The installer normally sets the `PATH`.
+
+## Obtain data from an app
+
+If an app is capable of responding to requests, you can obtain the following data from the app using terminal inline middleware:
```

Comment:
@Tratcher How useful do you feel this data is for troubleshooting?

Reply:
It depends on the issue. It's great for debugging forwarders, config, etc..